src/analyzer/protocol/dns/DNS.h

```diff
@@ -57,19 +57,24 @@ enum RR_Type {
 	TYPE_TKEY = 249,	///< Transaction Key (RFC 2930)
 	TYPE_TSIG = 250,	///< Transaction Signature (RFC 2845)
 	TYPE_CAA = 257,		///< Certification Authority Authorization (RFC 6844)
+	TYPE_SSHFP = 44,	///< SSH Public Key Fingerprint (RFC 4255)
```

Comment:
Can you insert these into the above in numeric order?

Reply:
Done :)